### PR TITLE
Enable ironic testing in component lines

### PR DESCRIPTION
**Pull request by lewisdenny**
Update podified-multinode-edpm-deployment-crc-bootstrap-staging-ironic to configure networking to test Ironic.

---
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/2494

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/2483

### CODE REVIEW

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

As dependent patches have updated

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -2,108 +2,5 @@
...
-      - cifmw-molecule-switch_config
-      - cifmw-molecule-tempest
-      - cifmw-molecule-test_deps
-      - cifmw-molecule-test_operator
-      - cifmw-molecule-tofu
-      - cifmw-molecule-update
-      - cifmw-molecule-update_containers
-      - cifmw-molecule-validations
-      - cifmw-molecule-virtualbmc
-    github-experimental-trigger:
```
Comment (frenzyfriday) [inline zuul.d/projects.yaml:99]: are we disabling all the check jobs and the downstream trigger jobs?

Comment (pablintino) [reply zuul.d/projects.yaml:99]: @frenzyfriday I guess this is a leftover from @lewisdenny testing.
@lewisdenny, please, remove this file from the PR.

**Review by pablintino — CHANGES_REQUESTED** [review]

File: zuul.d/projects.yaml (hunk 1 above)
Comment (lewisdenny) [reply zuul.d/projects.yaml]: Sorry about that one guys, checked out the file from main now.

**Comment by lewisdenny** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Pull request merged**